src/examples/zkapps/dex/happy-path-with-proofs.ts

```diff
@@ -0,0 +1,118 @@
+import { isReady, Mina, AccountUpdate, UInt64 } from 'snarkyjs';
+import { createDex, TokenContract, addresses, keys } from './dex.js';
+import { expect } from 'expect';
+import { tic, toc } from '../tictoc.js';
+
+await isReady;
+let doProofs = true;
+
+tic('Happy path with proofs');
+console.log();
+
+let Local = Mina.LocalBlockchain({ proofsEnabled: doProofs });
```

Comment:
I noticed most token examples exceed the new account update limits, so just FYI if this also is the case here we might have to add `enforceTransactionLimits: false`